doc/v2.md

```diff
@@ -105,6 +111,8 @@ Get to know how [paginators work here](./paginators.md).
 **Arguments**:
   - `query: string`
   - `options?: Tweetv2SearchParams`
+or
+  - `options?: Tweetv2SearchParams`
```

Comment:
<img width="379" alt="image" src="https://user-images.githubusercontent.com/4018426/161946584-a3685c5d-5b1e-4778-a649-a47e7029476b.png">

Not super clear

Reply:
Yep, I should add a line break between first options and `or`